src/Badges.sol

```diff
@@ -272,11 +275,15 @@ contract Badges is
     tokenURIs[tokenId] = _uri;
 
     emit Transfer(address(0), _to, tokenId);
-
+    emit Locked(tokenId);
     specDataHolder.setBadgeToRaft(tokenId, raftTokenId);
     return tokenId;
   }
 
+  function locked(uint256) public pure returns (bool) {
+    return true;
```

Comment:
This should return true only if the badge exists, no? For example, a burned badge or a non-existant badge will also return true here. 

Can you add test coverage to these 2 cases, please?

Reply:
fixed